courses/free-courses-hi.md

```diff
@@ -356,6 +356,7 @@
 * [Complete Git and GitHub Tutorial](https://www.youtube.com/watch?v=apGV9Kg7ics&t=7s) - Kunal Kushwaha
 * [Complete Git and Github Tutorials for Beginners](https://www.youtube.com/watch?v=Ez8F0nW6S-w) - Apna College
 * [Complete Git and GitHub Tutorials For Beginners In Hindi](https://www.youtube.com/playlist?list=PLzdlNxYnNoafZq1AKcqiGvj0gkzrjmgq7) - Code House
+* [Complete Git in one video](https://youtu.be/tz82ola3oy0?si=hez8NF3ZqatztrdR) - Hitesh choudhary
```

Comment:
```suggestion
* [Complete Git in one video](https://www.youtube.com/watch?v=tz82ola3oy0) - Hitesh choudhary
```

Reply:
Update your url with this one we don`t accept shortened youtube url links thank you.